R/main.R

```diff
@@ -31,11 +37,17 @@ test_main <- function(database_path = Sys.getenv('DATABASE_PATH'),
                       initial_aon_offset = Sys.getenv('INITIAL_AON_OFFSET'),
                       aon_recurring_interval = Sys.getenv('AON_RECURRING_INTERVAL'),
                       email) {
+  
+  annual_report_time <-  as.numeric(annual_report_time)
+  initial_aon_offset <-  as.numeric(initial_aon_offset)
+  aon_recurring_interval <-  as.numeric(aon_recurring_interval)
+  
   # Change email to testing email 
   db <- import_awards_db(database_path)
   db$pi_email <- email
   utils::write.csv(db, database_path, row.names = FALSE)
   
+  #runs fine individually but has issues when run from the top
```

Comment:
Is this a bug comment?  If so, let's make a Github issue.

Reply:
I'll remove it!  This was an issue I had running it locally on my Mac that I made a note to myself when initially was getting to know the bot. I must have committed it by accident.